UI/NativeApp.cpp

```diff
@@ -879,6 +888,11 @@ void NativeShutdownGraphics() {
 	winAudioBackend = nullptr;
 #endif
 
+#ifdef _WIN32
+	delete winCamera;
```

Comment:
if (winCamera) ?

Reply:
Right.

Reply:
Not really necessary as delete handles nullptr, but it's fine either way I think.